pull-kueue-verify-main and pull-kueue-test-e2e-kueueviz-main are failing due to cypress download

**What happened**:

failures: https://github.com/kubernetes-sigs/kueue/pull/7707#issuecomment-3548206256

**What you expected to happen**:
no such failures
**How to reproduce it (as minimally and precisely as possible)**:
build on ci
**Anything else we need to know?**:
```
#8 9.922 npm error [FAILED] Failed downloading the Cypress binary.
#8 9.922 npm error [FAILED] Response code: 500
#8 9.922 npm error [FAILED] Response message: Internal Server Error
#8 9.925 npm error A complete log of this run can be found in: /root/.npm/_logs/2025-11-18T15_32_27_911Z-debug-0.log
#8 ERROR: process "/bin/sh -c npm install" did not complete successfully: exit code: 1
------
 > [4/5] RUN npm install:
9.922 npm error Installing Cypress (version: 15.6.0)
9.922 npm error
9.922 npm error [STARTED] Task without title.
9.922 npm error Failed downloading the Cypress binary.
9.922 npm error Response code: 500
9.922 npm error Response message: Internal Server Error
9.922 npm error [FAILED] Failed downloading the Cypress binary.
9.922 npm error [FAILED] Response code: 500
9.922 npm error [FAILED] Response message: Internal Server Error
9.925 npm error A complete log of this run can be found in: /root/.npm/_logs/2025-11-18T15_32_27_911Z-debug-0.log
------
Dockerfile:10
--------------------
   8 |     
   9 |     # Install ALL dependencies (including dev dependencies needed for build)
  10 | >>> RUN npm install
  11 |     
  12 |     # Copy source files
--------------------
ERROR: failed to build: failed to solve: process "/bin/sh -c npm install" did not complete successfully: exit code: 1
make: *** [Makefile:270: npm-depcheck] Error 1
+ EXIT_VALUE=2
+ set +o xtrace
```

--- a/Makefile-test.mk
+++ b/Makefile-test.mk
@@ -281,8 +281,7 @@ test-e2e-kueueviz-local: setup-e2e-env ## Run end-to-end tests for kueueviz with
 .PHONY: test-e2e-kueueviz
 test-e2e-kueueviz: setup-e2e-env ## Run end-to-end tests for kueueviz without running kueue tests.
 	@echo Starting kueueviz end to end test in containers
-# 	Temporarily disabling the cypress-based tests due to https://github.com/kubernetes-sigs/kueue/issues/7730
-# 	CYPRESS_SCREENSHOTS_FOLDER=$(ARTIFACTS)/cypress/screenshots CYPRESS_VIDEOS_FOLDER=$(ARTIFACTS)/cypress/videos \
-# 	ARTIFACTS=$(ARTIFACTS) KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) PROJECT_DIR=$(PROJECT_DIR)/ \
-# 	KIND_CLUSTER_FILE="kind-cluster.yaml" IMAGE_TAG=$(IMAGE_TAG) \
-# 	${PROJECT_DIR}/hack/e2e-kueueviz-backend.sh
+	CYPRESS_SCREENSHOTS_FOLDER=$(ARTIFACTS)/cypress/screenshots CYPRESS_VIDEOS_FOLDER=$(ARTIFACTS)/cypress/videos \
+	ARTIFACTS=$(ARTIFACTS) KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) PROJECT_DIR=$(PROJECT_DIR)/ \
+	KIND_CLUSTER_FILE="kind-cluster.yaml" IMAGE_TAG=$(IMAGE_TAG) \
+	${PROJECT_DIR}/hack/e2e-kueueviz-backend.sh

